WriterDesk1/backend/config.py

```diff
@@ -7,7 +7,7 @@ class Config(object):
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     CACHE_TYPE = "SimpleCache"
     CACHE_DEFAULT_TIMEOUT = 300
-    FEEDBACKVERSION = 0
+    FEEDBACKVERSION = str(0.01)
```

Comment:
waarom stringifyen we dit?

Reply:
en moet t dan ook niet zo in conftest

Reply:
in conftest doe ik het implicit. En zodat je 1.00 kan storen bijvoorbeeld en dat niet een 1 wordt